docker/docker-compose.development.yml

```diff
@@ -0,0 +1,22 @@
+version: "3"
```

Comment:
@djabarovgeorge added a development override file for easier testing. 

```bash
docker-compose -f docker-compose.yml -f docker-compose.development.yml up
```

Reply:
Great, it will be really handy 💪